### PR TITLE
Bug#277 fix: Backend>>The title of form view of Types is not appropriate.

### PR DESCRIPTION
**Pull request by apghalsasi25**

### CODE REVIEW

File: src/components/com_tjucm/administrator/views/type/view.html.php [hunk 1]
```diff
@@ -64,6 +64,8 @@ protected function addToolbar()
 
 		$user  = JFactory::getUser();
 		$isNew = ($this->item->id == 0);
+		$input           = JFactory::getApplication()->input;
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/administrator/views/type/view.html.php]: @apghalsasi25 remove unused code

File: src/components/com_tjucm/administrator/views/types/view.html.php [hunk 2]
```diff
@@ -67,8 +67,11 @@ protected function addToolbar()
 	{
 		$state = $this->get('State');
 		$canDo = TjucmHelper::getActions();
+		$input = JFactory::getApplication()->input;
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/administrator/views/types/view.html.php]: @apghalsasi25 remove unused code

File: src/components/com_tjucm/languages/administrator/en-GB/en-GB.com_tjucm.ini [hunk 3]
```diff
@@ -58,6 +59,14 @@ COM_TJUCM_TYPE_IMPORT_INVALID_FILE_CONTENT_ERROR="File content is invalid"
 COM_TJUCM_TYPE_IMPORT_SUCCESS_MSG="UCM type(s) imported successfully"
 COM_TJUCM_DUPLICATE_TYPE_ERROR="UCM type already exists"
 
+COM_TJLMS_COMPONENT="LMS"
```
Comment (ankush-maherwal) [inline src/components/com_tjucm/languages/administrator/en-GB/en-GB.com_tjucm.ini]: @apghalsasi25 why LMS and JTicketing related constants included in UCM?

Comment (apghalsasi25) [reply src/components/com_tjucm/languages/administrator/en-GB/en-GB.com_tjucm.ini]: The titles for those components were selected by switch case where these  language constant were used. but now it is changed. I will remove them.

**Pull request closed without merging**